public/pages/Monitors/containers/Monitors/Monitors.js

```diff
@@ -332,21 +332,21 @@ export default class Monitors extends Component {
       monitorIds,
     };
 
-    const { httpClient, notifications } = this.props;
+    const { httpClient, notifications, history } = this.props;
 
-    const response = await httpClient.get('../api/alerting/alerts', { query: params });
-
-    if (response.ok) {
-      const { alerts, totalAlerts } = response;
-      this.setState({
-        alerts,
-        totalAlerts,
-        showAcknowledgeModal: true,
-      });
-    } else {
-      console.error(response);
-      backendErrorNotification(notifications, 'get', 'alerts', response.err);
-    }
+    const response = await getAlerts({
+      params,
+      httpClient,
+      notifications,
+      location: this.props.location,
```

Comment:
Minor comment: To better align with how the `notifications`, `history`, and `httpClient` parameters are initialized, [line 335](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/274/files?diff=unified&w=0#diff-7e1c541b82817549d3c59774fa18474aad37608a46ff1f16057cd1ab5e0dd12eR335) could be refactored to `const { httpClient, notifications, history, location } = this.props;`, and this line could be refactored from `location: this.props.location,` to just `location,`.